internal/clientsholder/clientsholder.go

```diff
@@ -55,6 +55,7 @@ type ClientsHolder struct {
 	K8sNetworkingClient  networkingv1.NetworkingV1Interface
 	MachineCfg           ocpMachine.Interface
 	MergedKubeConfigFile string
+	KubeConfig           []byte
```

Comment:
Do we need to keep the two possibilities for storing the `KubeConfig`?

Reply:
But the temp file already contains the `KubeConfig`. Can't you just read it when it's needed?

Reply:
Okay my latest commit https://github.com/test-network-function/cnf-certification-test/pull/759/commits/7a3e41e4832c0f9e5d12945479d09b46c82fc64d has removed the work done by @greyerof because it was unused.  The preflight suite now just uses a byte array param instead of requiring the file to be present.